src/main/java/org/thoughtcrime/securesms/connect/KeepAliveService.java

```diff
@@ -29,7 +29,7 @@ public static void maybeStartSelf(Context context) {
         // note, that unfortunately, the check for isIgnoringBatteryOptimizations() is not sufficient,
         // this checks only stock-android settings, several os have additional "optimizers" that ignore this setting.
         // therefore, the most reliable way to not get killed is a permanent-foreground-notification.
-        if (Prefs.isNotificationsEnabled(context) && Prefs.reliableService(context))  {
+        if (Prefs.reliableService(context))  {
```

Comment:
k, so we do not check for notification state any longer (we could check if notifications are enabled in some account) but start the `KeepAliveService` unconditionally.

probably fine, as also the option is now independent, also that makes code and states simpler in general.

Reply:
yeah, also simpler for the user, if they enable the reliable background service/permanent notification, it is harder to understand if it is only enable only under certain circumstances like "at least one account have notifications enabled" and since it is a global option it is easier to enable/disable it directly instead of relaying in disabling notifications for all accounts